src/BPEssentials/Utils/LevenshteinDistanceUtil.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+
+namespace BPEssentials.Utils
+{
+    public static class LevenshteinDistanceUtil
```

Comment:
I suggest changing this to `LevenshteinDistance`.
```suggestion
    public static class LevenshteinDistance
```